pulumi/integration_tests/__main__.py

```diff
@@ -1,30 +1,57 @@
+import json
 import os
 import sys
 from pathlib import Path
 
 sys.path.insert(0, "..")
 
-from typing import Optional, cast
+from typing import Mapping, Optional, cast
 
 import pulumi_aws as aws
 import pulumi_nomad as nomad
 from infra import config
 from infra.autotag import register_auto_tags
-from infra.docker_image_tag import version_tag
+from infra.docker_images import DockerImageIdBuilder, version_tag
 from infra.nomad_job import NomadJob, NomadVars
 from infra.quiet_docker_build_output import quiet_docker_output
 
 import pulumi
 
 
+def _container_images(
+    artifacts: Mapping[str, str], require_artifact: bool = False
+) -> str:
+    """
+    Build a map of "task name -> docker image URL".
+    See the .nomad file's var `container_images` for further documentation.
+    """
+    img_id_builder = DockerImageIdBuilder(config.container_repository())
+
+    def build_img_id_with_tag(image_name: str) -> str:
+        # A shortcut to grab the version tag from the artifacts map and build a
+        # Cloudsmith docker image url out of it.
```

Comment:
We're not _necessarily_ building a Cloudsmith image identifier here. Also, there's actually nothing Cloudsmith-specific about any of this logic.

Reply:
yep, this comment is a relic of when this class was called CloudsmithUrlBuilder :)